### DISCUSSION TITLE
How to prevent over-fitting?

### DISCUSSION THREAD

**Discussion by Hegelim** [PaddlePaddle/PaddleOCR#12639] [Q&A]
## Background
- I am fine-tuning on an existing pre-trained recognition model - arabic_PP-OCRv3_rec [here](https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.6/doc/doc_en/models_list_en.md#23-multilingual-recognition-modelupdating).
- The size of my training data is 1.2 million
- I modified a few places here and there in the config file. For privacy concerns, here are a few places that I changed:

Because of the volume of data, there are 18750 iters of each training epoch, so I set 
```
eval_batch_step: [0, 18750]
```

I changed the optimizer to use piecewise learning strategy
```
Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  lr:
    name: Piecewise
    decay_epochs:
    - 5
    values:
    - 5.0e-05
    - 1.0e-05
  regularizer:
    name: L2
    factor: 3.0e-05
```

For data augmentation, I deleted `RecConAug` completely because just combining 2 same images together sounds weird to me. I also used the following data augmentation
```
    - RecAug:
        noise_prob: 0.4
        jitter_prob: 0.4
        blur_prob: 0.4
        hsv_aug_prob: 0.4
```
In terms of batch size, I set 64 for both train and eval 

For everything else, I left it as is.

## Issue
I noticed the training accuracy went up really quickly. Just before the 1st epoch is finished, the acc is already near 97%. 
```
[2023/09/22 23:54:25] ppocr INFO: epoch: [1/100], global_step: 17800, lr: 0.000050, acc: 0.968750, norm_edit_dis: 0.991450, CTCLoss: 0.157956, SARLoss: 0.285718, loss: 0.451693, avg_reader_cost: 0.00020 s, avg_batch_cost: 0.43793 s, avg_samples: 64.0, ips: 146.14070 samples/s, eta: 9 days, 12:45:49
```

## Question
- why does the accuracy go up so fast? I have not seen such behavior in previous experiements.
- Looks to me this is over-fitting. What are some other options to prevent over-fitting? I tried to use a lot of data already. 

## System
- 系统环境/System Environment：ubuntu 20.04
- 版本号/Version：Paddle：  PaddleOCR： paddleocr 2.6.1.0 paddlepaddle-gpu 2.4.2
